packages/shorebird_cli/lib/src/shorebird_android_artifacts.dart

```diff
@@ -16,8 +16,10 @@ class MultipleArtifactsFoundException implements Exception {
 
   @override
   String toString() {
+    final artifacts = List<FileSystemEntity>.from(foundArtifacts)
+      ..sort((a, b) => a.path.compareTo(b.path));
```

Comment:
`package:collection` has a `sortedBy` function that might make this a bit cleaner:

```dart
final sortedArtifacts = foundArtifacts.sortedBy((a) => a.path);
```